source/parser/comptime.civet

```diff
@@ -0,0 +1,123 @@
+"civet coffeePrototype"
+
+import type {
+  ComptimeExpression
+  ComptimeStatement
+  StatementTuple
+} from ./types.civet
+
+import {
+  gatherRecursive
+} from "./traversal.civet"
+
+{ getInitialConfig } from "../parser.hera"
+
+generate, { prune, type Options } from "../generate.civet"
+
+function processComptime(statements: StatementTuple[]): void
+  // Prevent comptime setting from being overridden by directives
+  // by looking at initialConfig instead of config
+  return unless getInitialConfig()?.comptime
```

Comment:
If `comptime` is disabled does that mean that the expression is evaluated at runtime?

Reply:
Yes. I figured that was a reasonable fallback. In particular, it gives a chance for correct automatic typing in VSCode. I should probably add mention of this to the documentation though.

I also wondered whether a warning (or maybe a comment in the code output?) might be appropriate. I've seen some comptime systems add `/* compiled on ...date... */`. Could also plausibly add `/* comptime disabled during compilation */` or something to help debug.